docs/doc/10-deploy/02-deploying-databend.md

```diff
@@ -155,22 +170,29 @@ a. Create a folder named `databend` in the directory `/usr/local`.
 b. Download and extract the latest Databend release for your platform from [Github Release](https://github.com/datafuselabs/databend/releases):
 
 <Tabs groupId="operating-systems">
-<TabItem value="linux" label="Linux">
+<TabItem value="mac-x86_64" label="MacOS(x86)">
```

Comment:
It would be better to order the targets as:
Linux(x86)/Linux(arm)/MacOS(x86)/MacOS(arm)